lib/shopify_cli/github/issue_url_generator.rb

```diff
@@ -2,17 +2,22 @@ module ShopifyCLI
   module GitHub
     module IssueURLGenerator
       def self.error_url(error)
-        title = "#{error.class}: #{error.message}"
+        title = "[Bug]: #{error.class}: #{error.message}"
         labels = "type:bug"
-        content = File.read(File.join(ShopifyCLI::ROOT, ".github/ISSUE_TEMPLATE.md"))
 
         # take at most 5 lines from backtrace
         stacktrace = error.backtrace.length < 5 ? error.backtrace : error.backtrace[0..4]
```

Comment:
Why do we limit stack trace to 5 lines? Seems like a "more the merrier" situation.

Reply:
@molly-yu I see you implemented this in https://github.com/Shopify/shopify-cli/pull/1654

Do you recall the reason the stacktrace was limited to 5 lines?

Reply:
If I remember it correctly, Molly might have limited the length of the stack trace due to character limits in the URL parameter, because that information is passed as part of the **Submit a bug report** link from the CLI when an error is encountered.